rpc/backend/blocks.go

```diff
@@ -495,7 +545,8 @@ func (b *Backend) EthBlockFromTendermintBlock(
 
 	ethHeader := rpctypes.EthHeaderFromTendermint(block.Header, bloom, baseFee)
 	msgs := b.EthMsgsFromTendermintBlock(resBlock, blockRes)
-
+	msgs2 := b.EthMsgsFromTendermintEndBlock(blockRes)
```

Comment:
what is difference between msgs and msgs2?

Reply:
https://github.com/omni-network/ethermint/pull/3#discussion_r1272289927